### DISCUSSION TITLE
Welcome to Algorithms-for-Automated-Driving Discussions!

### DISCUSSION THREAD

**Discussion by thomasfermi** [thomasfermi/Algorithms-for-Automated-Driving#1] [General]
## 👋 Welcome!
  This is a place where you can connect with me (the book's author) and with other readers/students. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas on how the book could evolve and improve.
  * Engage with other readers.
  * Welcome others and are open-minded. 

  To get started, you can comment below with an introduction of yourself and tell us about your experience with the book.

**Comment by aerojunkie** [comment]
Hello, I just wanted to let you know that I really like this book! I'm building a website that will (eventually) allow people to organize links to their favorite resources into learning journeys. It's mostly for automation and control topics at the moment. I've added a link to this Jupyter book on the site so that more people can hopefully find it, https://resourcium.org/resource/algorithms-automated-driving.  I hope you are ok with this. If not, please let me know and I'll remove it.  Thanks for created such a wonderful book!  --Brian Douglas

**Reply by thomasfermi** [reply]
Hi @aerojunkie, thank you very much for the kind words! It means a lot to me, since I know your YouTube channel and found your videos really great! You might have seen that I also link to your videos in the PID chapter, and the reason I used drone height control as an example for PID was that I had seen this in one of your videos ;)

Thanks for adding a link to my book on your web site. I am more than ok with this :) The concept of the website sounds really interesting! I suggested a link regarding model predictive control to your web site just now :)
